README.md

```diff
@@ -17,8 +27,57 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+## Installation
+
+To use the **JupyterHub Pytest Plugin**, you will first need to install it using pip by either:
+
+- installing it locally
+  ```bash
+  pip install --upgrade pip
+  pip install git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
+- adding it to the `requirements.txt` file of your project:
+
+  ```python
+  # in requirements.txt
+  git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
+
+  then run:
+
+  ```bash
+  pip install --upgrade pip
+  pip install -r requirements.txt
+  ```
+
+## Usage
+
+To use a specific fixture or mock, import it from its module in the `pytest_jupyterhub` package:
+
+```python
+from pytest_jupyterhub.jupyterhub_spawners import hub_app
+```
```

Comment:
The pytest docs describe differently how to use a plugin in a test suite: https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file

Is there a reason why we're taking a different approach here and importing the fixtures same like we'd do from a regular Python package?

Reply:
Hmm, true. I might need to change the `DockerSpawner` [draft PR](https://github.com/jupyterhub/dockerspawner/pull/476) as well since I imported it instead of loading the plugin. Thank you for catching this.